optashift-employee-rostering-webapp/pom.xml

```diff
@@ -46,6 +46,50 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-war-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>war-exploded</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>exploded</goal>
+            </goals>
+            <configuration>
+              <webappDirectory>${wildfly.home}/standalone/deployments/${project.build.finalName}.war</webappDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-resources</id>
+            <!-- here the phase you need -->
```

Comment:
Most comments are good: those that explain complexity, non-obvious things and when some code is a workaround or hack (which information when that will become obsolete).
Comments that explain what the code already explains, or what the javadoc/maven pom spec already explains, aren't useful. Let's remove this line.